### DISCUSSION TITLE
Check out my pcb layout, im a noob!

### DISCUSSION THREAD

**Discussion by Babylonrising** [shanteacontrols/OpenDeck#284] [Q&A]
Hey guys! I'm a TOTAL noob with this stuff with the exception of building a bunch of eurorack modules, I noticed to use encoders up to 8 at a time are mapped to two single pins ie R0 and R1 go to 8 different encoders legs, can you check this pcb I'm making as a breakout that holds 12 regular encoders and then 4 push button encoders (didn't do those yet) does this look correct? This is just one section of a large midi controller I'm designing for myself that I'll be sharing the build on here as I go
![encoder breakout pcb](https://github.com/shanteacontrols/OpenDeck/assets/165958798/96d1c696-18fe-4861-9563-7f7a28c75f1e)

**Comment by paradajz** [comment]
On the L board the center pin doesn't go to ground but column instead. Please read the wiki carefully, everything is documented there.
https://github.com/shanteacontrols/OpenDeck/wiki/Official-OpenDeck-L-boards#rotary-encoders

**Reply by Babylonrising** [reply]
So i went back and revised the encoder pcb section, does this look right? first 12 are regular encoders and last 4 are push encoders
![encoder breakout v2](https://github.com/shanteacontrols/OpenDeck/assets/165958798/fef146bb-76c7-44ac-8a74-b1607720411b)
